jujugui/static/gui/src/app/components/user-profile/_user-profile.scss

```diff
@@ -1,7 +1,67 @@
 .user-profile {
+
     &__create-new {
         float: right;
         font-size: 1rem;
+        width: 400px;
+        text-align: right;
+
+        &.collapsed {
+            .deployment-input__field,
+            .button--inline-neutral.second {
+                width: 0;
+                border-width: 0;
+            }
+            .deployment-input__field {
+              padding: 0;
+            }
+            .button--inline-neutral.first {
+                width: 150px;
+                border-width: 1px;
+            }
+        }
+
+        .button--inline-neutral.first {
+            width: 0;
+            border-width: 0;
+        }
+
+        .button--inline-neutral {
+            overflow: hidden;
+            white-space: nowrap;
+            transition: all 0.5s ease-in;
+            width: 150px;
+            outline: none;
+            padding: 10px 0;
+            display: inline-block;
+        }
+
+        .deployment-input {
+            display: inline-block;
+
+            &.error {
+                .deployment-input__field {
+                    border: 1px solid #df382c;
+                }
+            }
+        }
+
+        .deployment-input__errors {
+            position: absolute;
+            z-index: 600;
```

Comment:
Could you please use a z-index from our list: https://github.com/juju/juju-gui/blob/develop/jujugui/static/gui/src/app/assets/css/base.scss#L34

e.g. https://github.com/juju/juju-gui/blob/develop/jujugui/static/gui/src/app/assets/css/_header.scss#L13